helix-view/src/document.rs

```diff
@@ -101,6 +105,8 @@ pub struct Document {
     syntax: Option<Syntax>,
     /// Corresponding language scope name. Usually `source.<lang>`.
     pub(crate) language: Option<Arc<LanguageConfiguration>>,
+    /// Sets if syntax highlighting should be used or not
+    pub highlight_syntax: bool,
```

Comment:
Should be removed and set the language to None

Reply:
Currently, it is not an issue, but I think that setting language to `None` when disabling parser, may cause some confusion to someone in the future. Language by itself is not changing/non-existent, it is just not parsed.